src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/tests/registries/ToolbarActionRegistry.test.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import toolbarActionRegistry from '../../registries/ToolbarActionRegistry';
+import AbstractToolbarAction from '../../toolbarActions/AbstractToolbarAction';
+
+jest.mock('../../../../services/Initializer', () => jest.fn());
+jest.mock('../../toolbarActions/DeleteToolbarAction', () => jest.fn());
+
+beforeEach(() => {
+    toolbarActionRegistry.clear();
+});
+
+test('Clear all adapters', () => {
```

Comment:
We are not adding `adapters` but `toolbarActions` here... That's true for all test cases. Have you copied it from the other `ToolbarActionRegistry` test? Can you fix it there too?